integration/nwo/network.go

```diff
@@ -704,6 +700,91 @@ func (n *Network) Bootstrap() {
 	n.ConcatenateTLSCACertificates()
 }
 
+func (n *Network) createDockerNetwork() {
+	_, err := n.DockerClient.CreateNetwork(
+		docker.CreateNetworkOptions{
+			Name:   n.NetworkID,
+			Driver: "bridge",
+		},
+	)
+	Expect(err).NotTo(HaveOccurred())
+
+	if runtime.GOOS == "darwin" {
+		n.checkDockerNetworks()
+	}
+}
+
+// checkDockerNetworks attempts to discover if the docker network configuration
+// will prevent a container from accessing the host. This commonly happens when
+// using Docker for Mac on home networks because most home routers provide DHCP
+// addresses from 192.168.1.0/24 and the default Docker daemon config uses
+// 192.168.0.0/20 as one of the default local address pools.
+//
+// https://github.com/moby/libnetwork/blob/1a17fb36132631a95fe6bb055b91e24a516ad81d/ipamutils/utils.go#L18-L20
+//
+// Docker can be configured to use different addresses by addding an
+// appropriate default-address-pools configuration element to "daemon.json".
+//
+// For example:
+//   "default-address-pools":[
+//       {"base":"172.80.0.0/16","size":24},
+//       {"base":"172.81.0.0/16","size":24}
```

Comment:
Just a nit, but wouldn't it be better to pick from the officially reserved ranges just in case someone looks at the comment?
```suggestion
//       {"base":"172.16.0.0/16","size":16},
//       {"base":"172.17.0.0/16","size":16}
```
but I guess those are in the default.  You can also override the bridge network with the "bip" config option but again I guess this is just a comment

Reply:
> Just a nit, but wouldn't it be better to pick from the officially reserved ranges just in case someone looks at the comment?

They're from the reserved range of 172.16.0.0/12. So, I'm not sure why it's wrong?

Reply:
You are correct and my math is not. I'll address this with the spelling fix.